functions-and-operators/string-functions.md

```diff
@@ -173,10 +173,114 @@ Synonym for `LOWER()`.
 
 Return the leftmost number of characters as specified.
 
+* usage: LEFT(`str`, `len`)
+    * len: length of character to return
+    * str: original string to extract characters
+* if `len` is equal or less than 0, the function returns empty string
+* function is multibyte safe.
+* If any argument is `NULL`, the function returns `NULL`.
+
```

Comment:
```suggestion
* If `len` is greater than the length of `str`, the function returns `str`.

```

Reply:
How about adding an explanation for `SELECT LEFT('ABCED', 6);`?

Reply:
thank you for the suggestion!